### PR TITLE
Added Automatic Location Search

### PR DESCRIPTION
**Pull request by kaushikjatin**
I have added automatic location search...for that I used the OpenCage API.
Line 91 in App.js -> Generate your own API key and then put in at that spot.
Use this link (https://www.youtube.com/watch?v=JdJ2VBbYYTQ) and time -> 1:33 to see how to generate API key.

### CODE REVIEW

**Comment by 1502shivam-singh** [comment]
Nice work @kaushikjatin. I was thinking of using the Google maps Geocoding API, still this seems alright too 👍.

Some things I wanted to know (just for research purpose)
- How did you get to know about this project (Social media, GitHub search or elsewhere)?
- Any thing you want to add in this. (Go ahead and make an issue if you feel so)

 Also, thanks for contributing and do share this with people you know, so that one's who are in need get it.

(*a star would be appreciated 😊*)

File: src/App.js [hunk 1]
```diff
@@ -84,6 +84,47 @@ export default function App() {
     }
   });
 
+  function success(pos) {
+    var crd = pos.coords;
+    const latitude=crd.latitude;
+    const longitude=crd.longitude;
+    var apikey = '91e34486d51b4b55b4b54c04c205ec80';
```
Comment (1502shivam-singh) [inline src/App.js]: I would prefer 'let' in place of 'var' for the reasons of block scoping and in general as a nice practice

File: src/App.js [hunk 2]
```diff
@@ -84,6 +84,47 @@ export default function App() {
...
+      + '?'
+      + 'key=' + apikey
+      + '&q=' + encodeURIComponent(latitude + ',' + longitude)
+      + '&pretty=1'
+      + '&no_annotations=1';
+    var request = new XMLHttpRequest();
+    request.open('GET', request_url, true);
+    request.onload = function() {
+      if (request.status === 200){ 
+        var data = JSON.parse(request.responseText);
```
Comment (1502shivam-singh) [inline src/App.js:103]: Ditto. As mentioned above and everywhere else it can be replaced without leading to bugs.

Comment (kaushikjatin) [reply src/App.js:103]: Ok Shivam , I will do the updates.
By the way I just searched for the label on Github, and got in some the initial results.

**Comment by 1502shivam-singh** [comment]
You could have just pushed to the same branch, then updates would be added, no need to close it and make a new one 
@kaushikjatin.

**Pull request closed without merging**